mlem/cli/config.py

```diff
@@ -32,9 +32,8 @@ def config_set(
     ),
 ):
     """Set configuration value
-
-    Examples:
-        $ mlem config set pandas.default_format csv
+    \n\n
+    Documentation: <https://mlem.ai/doc/command-reference/config>
```

Comment:
I don't think this is needed in docstring - you don't look for cli help in code. And if you do, you already know about docs site. Can we add this to help output dynamically?

Reply:
I also see that you didn't add those for every command - that is exactly why adding those dynamically is more reliable

Reply:
For cases where it is not `https://mlem.ai/doc/command-reference/{name}` we can add an override attribute to `MlemCommand`

Reply:
Lol you already did that. Then why also have it here?

Reply:
Ok thats instead of override attribute. I guess attribute will be a bit clearer and uniform, but I'll accept if you don't agree

Reply:
Nice talking to you @mike0sv !

Reply:
You too